x-pack/test/detection_engine_api_integration/security_and_spaces/update_prebuilt_rules_package/update_prebuilt_rules_package.ts

```diff
@@ -133,6 +134,8 @@ export default ({ getService }: FtrProviderContext): void => {
       // Verify that the _perform endpoint returns the same number of installed rules as the status endpoint
       // and the _review endpoint
       const installPrebuiltRulesResponse = await installPrebuiltRules(supertest);
+      await es.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES });
+
```

Comment:
Installed rules fetching chunk looks to be repeated. It could be a reusable function with `es.indices.refresh()` inside

```
const { body: rulesResponse } = await supertest
        .get(`${DETECTION_ENGINE_RULES_URL_FIND}?per_page=10000`)
        .set('kbn-xsrf', 'true')
        .send()
        .expect(200);
```